src/phase-management/phase-rules.json

```diff
@@ -0,0 +1,392 @@
+              "fact": "isPastScheduledStartTime",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isOpen",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "isClosed",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "isPredecessorPhaseClosed",
+              "operator": "equal",
+              "value": true
+            }
+          ]
+        },
+        "event": {
+          "type": "canOpen"
+        }
+      }
+    ],
+    "AppealsResponse": [
+      {
+        "name": "Appeals Response Open",
+        "conditions": {
+          "all": [
+            {
+              "fact": "isPastScheduledStartTime",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isOpen",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "isClosed",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "isPredecessorPhaseClosed",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isPostMortemOpen",
+              "operator": "notEqual",
+              "value": true
+            }
+          ]
+        },
+        "event": {
+          "type": "canOpen"
+        }
+      }
+    ]
+  },
+  "closeRules": {
+    "Registration": [
+      {
+        "name": "Registration Close",
+        "conditions": {
+          "all": [
+            {
+              "fact": "isOpen",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isPastScheduledEndTime",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isClosed",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "registrantCount",
+              "operator": "greaterThanInclusive",
+              "value": 1
+            }
+          ]
+        },
+        "event": {
+          "type": "canClose"
+        }
+      }
+    ],
+    "Submission": [
+      {
+        "name": "Submission Close",
+        "conditions": {
+          "all": [
+            {
+              "fact": "isClosed",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "isOpen",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "any": [
+                {
+                  "fact": "isPastScheduledEndTime",
+                  "operator": "equal",
+                  "value": true
+                },
+                {
+                  "all": [
```

Comment:
I couldn't understand why submission phase is closed here if this is F2F and there is a submission pending to be reviewed.

Reply:
Yeah this is an oversight - submission should not be closed! I'll fix it.